tests/Feature/FormMigration/Steps/TestDonationGoal.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Give\Tests\Feature\FormMigration\Steps;
+
+use Give\DonationForms\Properties\FormSettings;
+use Give\DonationForms\ValueObjects\GoalType;
```

Comment:
Let's remove these two first statements as they aren't being used in this class.

Reply:
resolved 09397bdd